esbuild.config.mjs

```diff
@@ -102,14 +102,25 @@ const serverBuild = {
 
 export { clientBuild, serverBuild }
 
+const executeTypeScriptCompile = () => {
+  console.log('🧬 [tsc] running TypeScript compiler...')
+  execSync(`npx tsc --diagnostics`, { stdio: 'inherit' })
+  console.log('✅ [tsc] done')
```

Comment:
Do we want these logs? :)

Reply:
who doesn't like emoji spam in their terminal?